### PR TITLE
Clean up: ISBN

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- (re)moved schema from miscellaneous to new folder called digitalIdentifier for better grouping and decluttering of the miscellaneous folder

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison or @Peyman-N feel free to merge if you are also happy with the changes

File: schemas/digitalIdentifier/ISBN.schema.tpl.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "identifier": {
       "type": "string",
       "pattern": "^([0-9]{3}-|)[0-9]{1}-[0-9]{3}-[0-9]{5}-[0-9]{1}$",
-      "_instruction": "Enter the International Standard Book Number of the International ISBN Agency."
+      "_instruction": "Enter the numeric commerical book identifier 'International Standard Book Number' (ISBN) following the defined pattern (e.g., 123-4-567-89012-3 (13-digit ISBN) or 4-567-89012-3 (10-digit ISBN))."
```
Comment (Peyman-N) [inline schemas/digitalIdentifier/ISBN.schema.tpl.json]: Typo `commercial`

**Review by lzehl — APPROVED** [review]

File: schemas/digitalIdentifier/ISBN.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/digitalIdentifier/ISBN.schema.tpl.json]: I've corrected the typo

**Comment by lzehl** [comment]
@Peyman-N or @aeidi89 or @apdavison feel free to merge if you also approve now

**Review by aeidi89 — APPROVED** [review]
Looks good.

**Pull request merged**